src/apis/index.js

```diff
@@ -100,6 +100,37 @@ export function fetchDevices(payload) {
   return ajax.get(url, payload, { shouldCamelizeKeys: false });
 }
 
+export async function getContributors() {
+  const contributorsWebLink =
```

Comment:
There are otherr SUSI.AI repos as well like `susi_linux`, etc. Please make a list of all active SUSI.AI repos and then show contributors in all of them.

Also, here code is being repetative. Avoid that. Also when you need to make multiple calls, use `Promise.all`